toolbox

```diff
@@ -875,15 +891,14 @@ init_container()
         fi
     fi
 
+
     if ! id -u "$init_container_user" >/dev/null 2>&3; then
-        if [ "$(readlink /home)" = var/home ] 2>&3; then
+        if  $init_container_home_link ; then
             # shellcheck disable=SC2174
-            if ! rmdir /home 2>&3 \
-                 && mkdir --mode 0755 --parents /var/home 2>&3 \
-                 && ln --symbolic var/home /home 2>&3; then
+            if ! (rmdir /home 2>&3 && ln --symbolic /var/home /home 2>&3); then
```

Comment:
It would be good to mention something about this in the commit message, or split it into a separate commit. Apparently this isn't the first time I have made this typo. :/

By the way, the `mkdir` command got lost.